setup.py

```diff
@@ -9,6 +9,9 @@
     from distutils.core import setup
 
 
+with open("README.rst", "r") as fh:
```

Comment:
Could you add a function to return this value instead of assigning it to a constant variable? @morenol

Reply:
I am not sure why, but it seems that is like an standard to do that at this way.

You can see that in the tutorial of pypi https://packaging.python.org/tutorials/packaging-projects/#uploading-your-project-to-pypi

And also in edx repos https://github.com/edx/edx-rest-api-client/blob/4c6abafd095e05d8574952801b998d921ae96868/setup.py#L8

Reply:
Ok, let's keep it as it is.